Workshop.Domain/Entities/Management/Client.cs

```diff
@@ -22,4 +22,9 @@ public Client(string name, Company company)
         CompanyId = company.Id;
         Company = company;
     }
+
+    public void AddRepresentative(Guid representativeId)
+    {
+        RepresentativeId = representativeId;
```

Comment:
Recebe o User como parâmetro e seta Representative e RepresentativeId baseado nele, para preencher os campos qnd for retornar